sdk/servicebus/azservicebus/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
I'm no attorney but this license is slightly different than the license in azcore and we should probably be consistent.

Reply:
(I'm assuming this is the license you're talking about, btw)

https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/LICENSE.txt

Reply:
Updated so the contents are the same.